packages/common/src/content/_internal/computeProps.ts

```diff
@@ -18,8 +19,11 @@ export const getItemExtent = (itemExtent: number[][]): IExtent => {
     : undefined;
 };
 
-export function deriveLocationFromItemExtent(itemExtent?: number[][]) {
-  const location: IHubLocation = { type: "custom" };
+export function deriveLocationFromItemExtent(
+  locationType: IHubLocationType,
+  itemExtent?: number[][]
+) {
```

Comment:
I don't think we need to pass in the `locationType` here, we'll always do `"item"`.

Also, I'm not certain if we will continue to convert to polygon here. I think the rectangle draw tool will expect a geometry of type extent.

I think we'll only convert to polygon for the "custom" option.